zerver/views/webhooks/jira.py

```diff
@@ -86,7 +90,9 @@ def convert_jira_markup(content, realm):
 def api_jira_webhook(request, user_profile, client,
                      payload=REQ(argument_type='body'),
                      stream=REQ(default='jira')):
+    # type: (HttpRequest, UserProfile, str, Dict[Any, Any], str) -> HttpResponse
```

Comment:
Also, `payload` is a Dict[str, Any] and stream should be a `text_type` (since it's unicode)